tests/system/Database/Live/PreparedQueryTest.php

```diff
@@ -114,8 +123,8 @@ public function testExecuteRunsQueryAndReturnsManualResultObject()
             return (new Query($db))->setQuery($sql);
         });
 
-        $query->execute('foo', 'foo@example.com', '');
-        $query->execute('bar', 'bar@example.com', '');
+        $query->execute('foo', 'foo@example.com', 'US');
+        $query->execute('bar', 'bar@example.com', 'GB');
```

Comment:
Why do you need to change the empty strings?

Reply:
Thanks, good comment.

This is because Oracle treats an empty string as null and the value of the country column is NotNull, so the query will fail.
oci8 also treats empty strings as null.

https://github.com/php/php-src/blob/master/ext/oci8/tests/bind_sqltafc.phpt#L149-L175

Reply:
You're correct.
Oracle treats an empty string as null.